src/main/java/com/KacFlor/ShopSpring/controller/CustomerController.java

```diff
@@ -22,8 +26,38 @@ public CustomerController(CustomerService customerService){
         this.customerService = customerService;
     }
 
```

Comment:
What about getCustomerById for admin?

Reply:
I have big problem with this and dont know how to do that because everytime it return null

Reply:
done